File: tests/testdata/codegen/simple-range-pp/nodejs/simple-range.ts
import * as pulumi from "@pulumi/pulumi";
import * as random from "@pulumi/random";

const numbers: random.RandomInteger[] = [];
for (const range = {value: 0}; range.value < 2; range.value++) {
    numbers.push(new random.RandomInteger(`numbers-${range.value}`, {
        min: 1,
        max: range.value,
        seed: `seed${range.value}`,
    }));
}
export const first = numbers[0].id;
export const second = numbers[1].id;

File: tests/testdata/codegen/simple-range-pp/python/simple-range.py
import pulumi
import pulumi_random as random

numbers = []
for range in [{"value": i} for i in range(0, 2)]:
    numbers.append(random.RandomInteger(f"numbers-{range['value']}",
        min=1,
        max=range["value"],
        seed=f"seed{range['value']}"))
pulumi.export("first", numbers[0].id)
pulumi.export("second", numbers[1].id)
